src/DIRAC/MonitoringSystem/Client/DataOperationSender.py

```diff
@@ -12,17 +12,20 @@
 from DIRAC.AccountingSystem.Client.Types.DataOperation import DataOperation
 from DIRAC.MonitoringSystem.Client.MonitoringReporter import MonitoringReporter
 
-sLog = gLogger.getSubLogger("__name__")
```

Comment:
please leave all these `sLog`.

But what should be used in the `__init__` is `self.log = sLog.getSubLogger` instead of `self.log = gLogger.getSubLogger`